packages/database/prisma/models/user.prisma

```diff
@@ -38,6 +38,8 @@ model User {
     feedReactions FeedItemReaction[]
     feedComments  FeedItemComment[]
     roles         UserRole[]
+
+    @@index(phoneNumber)
```

Comment:
I think `phoneNumber` index might not necessary because `phoneNumber` already have unique constraint which automatically got unique index too so please remove migration related to this index.

<img width="796" height="28" alt="Image" src="https://github.com/user-attachments/assets/bb6b5d09-e5ac-4ca8-9931-015cff965506" />